tests/foreman/ui/test_contentview.py

```diff
@@ -1035,6 +1037,125 @@ def test_positive_create_date_filter_rule_without_type(self):
+        start_date = '2010-01-01'
+        end_date = date.today().strftime('%Y-%m-%d')
+        content_view = entities.ContentView(
+            organization=self.organization).create()
+        product = entities.Product(organization=self.organization).create()
+        repo = entities.Repository(
+            product=product,
+            url=FAKE_9_YUM_REPO,
+        ).create()
+        repo.sync()
+        with Session(self) as session:
+            session.nav.go_to_select_org(self.organization.name)
+            session.content_views.add_remove_repos(
+                content_view.name, [repo.name])
+            session.content_views.add_filter(
+                content_view.name,
+                filter_name,
+                FILTER_CONTENT_TYPE['erratum by date and type'],
+                FILTER_TYPE['include'],
+            )
+            session.content_views.edit_erratum_date_range_filter(
+                content_view.name,
+                filter_name,
+                errata_types=['security'],
+                date_type=FILTER_ERRATA_DATE['issued'],
+                start_date=start_date,
+                end_date=end_date,
+                open_filter=False,
+            )
+            self.assertIsNotNone(self.content_views.wait_until_element(
+                common_locators['alert.success_sub_form']))
+            session.content_views.publish(content_view.name)
+            self.assertIsNotNone(self.content_views.wait_until_element(
+                common_locators['alert.success_sub_form']))
+            result = session.content_views.fetch_version_errata(
+                content_view.name, 'Version 1.0')
+            self.assertEqual(len(result), FAKE_9_YUM_SECURITY_ERRATUM_COUNT)
+            self.assertTrue(
+                all(
+                    errata[2] == FILTER_ERRATA_TYPE['security']
+                    for errata in result
+                )
+            )
+
+    @tier2
+    def test_positive_add_all_security_errata_id_filter(self):
+        """Create erratum filter to include only security errata and publish
+        new content view version
+
+        :id: bc0be8e8-af53-4db8-937d-93c49c937dcc
+
+        :BZ: 1275756
+
+        :CaseImportance: High
+
+        :expectedresults: all security errata is present in content view
+            version
+        """
+        filter_name = gen_string('alphanumeric')
+        content_view = entities.ContentView(
+            organization=self.organization).create()
+        product = entities.Product(organization=self.organization).create()
+        repo = entities.Repository(
+            product=product,
+            url=FAKE_9_YUM_REPO,
+        ).create()
+        repo.sync()
+        with Session(self) as session:
+            session.nav.go_to_select_org(self.organization.name)
+            session.content_views.add_remove_repos(
+                content_view.name, [repo.name])
+            session.content_views.publish(content_view.name)
```

Comment:
Previous test is rather a bonus - it's just a valid scenario which i've accidentally automated before noticing there should be different type of errata filter, so its flow follows just common sense.
And this test is automated in correspondence with steps from BZ to guarantee BZ coverage. I'm 90% sure that first publish has no effect on BZ reproduciveness, but just in case.